src/main/java/oshi/software/os/mac/local/SystemB.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) Daniel Widdis, 2015
+ * widdis[at]gmail[dot]com
+ * All Rights Reserved
+ * Eclipse Public License (EPLv1)
+ * http://oshi.codeplex.com/license
+ */
+package oshi.software.os.mac.local;
+
+import com.sun.jna.Library;
+import com.sun.jna.Native;
+import com.sun.jna.Structure;
+import com.sun.jna.ptr.IntByReference;
+import com.sun.jna.ptr.LongByReference;
+
+/**
+ * Memory obtained by vm_stat and sysctl
+ * 
+ * @author widdis[at]gmail[dot]com
+ */
+public interface SystemB extends Library {
```

Comment:
Well I'm continuing to add to that one as I go along so I don't think it's ready for submitting to a more widely used library...